tests/updater/controller/v1/test_routes.py

```diff
@@ -8,7 +8,7 @@
 
 async def _api_device_update(async_client, device, update_attribute, update_value):
     response = await async_client.patch(
-        f"/api/v1/devices",
+        f"/ui/bff/devices",
```

Comment:
Could be a but cleaner directly updating the device model - so that this test module has no dependency to the bff api. Optional.

Reply:
This seems a bit complicated just because of the way that endpoint functions, I think it introduces a lot of complexity directly into the tests that we may not want.  I think this should probably be left for a later PR in any case.